bundles/org.openhab.binding.siemenshvac/README.md

```diff
@@ -0,0 +1,128 @@
+# SiemensHvac Binding
+
+This binding is to support Siemens Hvac controller ecosystem, and the Web Gateway interface OZW672.
+A typical system is composed of:
+
+                                                    
+<=== Ethernet ===>   | OZW672 | <====== ¨BSB/LPB BUS ======> | Hvac Controler (RVS41.813/327) | ====== | Internal device in your system : sensors, boiler, external pac unit, ... |
+
+There's a lot of different Hvac controler depending on model in lot of different PAC constructor.
+Mine is a Atlantic Hybrid duo whith a Siemens RVS41.813/327 inside.
+
+Siemens have a complete set of controler reference under the name "Siemens Albatros".
+Here some picture of such device.
+You can also find this device in other type of heating system : boiler or solar based.
+
+![](doc/Albatros.jpg)
+
+You will find some information about the OZW672.01 gateway on siemens web site : 
+[OZW 672 Page]
+([https://hit.sbt.siemens.com/RWD/app.aspx?rc=FR&lang=fr&module=Catalog&action=ShowProduct&key=BPZ:OZW672.01)
+
+With this binding, you will be able :
+
+- To consult the different parameters of your system like temperature, current heating mode, water temperature, and many more.
+- Modify the functionning mode of your device : temperature set point, heating mode, and others.
+
+
+
+## Supported Things
+
+Support many different things as the thing type is handle by autodiscovery.
```

Comment:
I think this can be adapted depending on the outcome of the [thing type discussion](https://github.com/openhab/openhab-addons/pull/14263#discussion_r1598335493).

Reply:
Suggestion: If no Things list can be provided that would help with manual configuration, could we drop this entire chapter? The notes (e.g. about discovery and gateway model support/tests) could be merged into the next chapter: Discovery.

Reply:
So to be sure : your recommandation is two remove the chapter ## Supported Things, and move the usefull consideration inside the ## Discovery chapters ?

Reply:
I've removed the section.